components/menu/index.css

```diff
@@ -10,170 +10,661 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-200);
+  --spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-200);
+
+  --spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-200);
+
+  --spectrum-menu-item-description-font-size: var(--spectrum-font-size-100);
+
+  
+  --spectrum-menu-section-header-font-size: var(--spectrum-font-size-200);
+  --spectrum-menu-section-header-min-width: var(--spectrum-component-height-200);
+
+  /* TODO: eventual token name: */
+  /* --spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-edge-to-text-not-selected-large); */
+  /* TODO: remove below once above is defined */
+  --spectrum-menu-item-selectable-edge-to-text-not-selected: 38px;
+
+  --spectrum-menu-item-checkmark-height: 12px;
+  --spectrum-menu-item-checkmark-width: 12px;
+}
+
+.spectrum-Menu--sizeXL {
+  --spectrum-menu-item-min-height: var(--spectrum-component-height-300);
+  --spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-300);
+  --spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-300);
+  --spectrum-menu-item-label-font-size: var(--spectrum-font-size-300);
+  --spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-300);
+  
+  --spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-300);
+  --spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-300);
+  --spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-300);
+
+  --spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-300);
+
+  --spectrum-menu-item-description-font-size: var(--spectrum-font-size-200);
+
+
+  --spectrum-menu-section-header-font-size: var(--spectrum-font-size-300);
+  --spectrum-menu-section-header-min-width: var(--spectrum-component-height-300);
+
+  /* TODO: eventual token name: */
+  /* --spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-edge-to-text-not-selected-extra-large); */
+  /* TODO: remove below once above is defined */
+  --spectrum-menu-item-selectable-edge-to-text-not-selected: 45px;
+
+  --spectrum-menu-item-checkmark-height: 14px;
+  --spectrum-menu-item-checkmark-width: 14px;
+}
+
+/* TODO: temporary & can be removed once the tokens are defined in the core tokens package */
+.spectrum--large {
+  .spectrum-Menu--sizeS {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 34px;
+  }
+  .spectrum-Menu--sizeM {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 42px;
+  }
+  .spectrum-Menu--sizeL {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 47px;
+  }
+  .spectrum-Menu--sizeXL {
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: 54px;
+  }
+}
+
+.spectrum--light {
+  --spectrum-menu-item-background-color-default: rgba(0, 0, 0, 0); /* --spectrum-gray-900 */
```

Comment:
Great call-out! I actually dug into this a bit [in Slack](https://adobedesign.slack.com/archives/G0143NJRLKY/p1662563496552649), but, the short story is: we need token definitions at the `100` level for `transparent-white` and `transparent-black` which would provide a `0%` in the alpha.

Right now, I'd have to try to combine two tokens together, BUT, if I do so, the resulting rule isn't valid CSS. For instance:

```
--spectrum-menu-item-background-color: --spectrum-gray-900; /* gray-900 === rgb(0,0,0) */
--spectrum-menu-item-background-opacity-hover: 10%;
...

background-color: rgba(var(--spectrum-menu-item-background-color) var(--spectrum-menu-item-opacity-hover));
```

The above ends up resolving to `rgba(rgb(0, 0, 0) 10%)` unfortunately. Sooooo, until we get that token, we shall hard-code.

Reply:
Thanks for calling out the issue Patrick. Then hard-coding them for now to avoid those combination issues might be the right choice. 
I can mark this as resolved for now.